e2e/e2e.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -e
+function bail {
+	echo "$*"
+	exit 1
+}
+
+function cleanup_server {
+	echo "The Server Log follows:"
+	cat "build/server_log.txt"
+	# Killing 0 sends the signal to all processes in the current
+	# process group
+	kill $SERVER_PID
+}
+
+# Travis CI is super cool but also uses ancient OS images
+if [ -f "/usr/bin/python3.6" ]; then
+	export PYTHON_BINARY="/usr/bin/python3.6"
+else
+	export PYTHON_BINARY=`which python3`
+fi
+
+
+mkdir -p "e2e_data"
```

Comment:
This assumes that the script is called from the root folder of the repo. When using the script for manual testing it might be easier to be able to call the script from anywhere and then have the script cd to the scripts location here (e.g. using `${BASH_SOURCE[0]}`.

Reply:
Oh I didn't know about `${BASH_SOURCE[0]}` very handy, thanks.